### PR TITLE
Now uses HR-TIME as the basis

### PR DESCRIPTION
**Pull request by plehegar**
Aligning with https://github.com/w3c/hr-time/pull/7

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <head>
   <title>Performance Timeline</title>
   <meta charset='utf-8'>
-  <script src='//www.w3.org/Tools/respec/respec-w3c-common' async class='remove'></script>
+  <script src='http://www.w3.org/Tools/respec/respec-w3c-common' async class='remove'></script>
```
Comment (igrigorik) [inline index.html]: Can we make this HTTPS? Otherwise the spec is broken when accessed over https.

File: index.html [hunk 2]
```diff
@@ -169,11 +169,11 @@
       </section>
 
       <section>
-        <h2>The <dfn>Performance</dfn> interface</h2>
+        <h2>Extensions to the <dfn>Performance</dfn> interface</h2>
 
-        <p>The <a>Performance</a> interface hosts performance related attributes and methods used to retrieve the performance metric data from the <a>Performance Timeline</a>.</p>
+        <p>This extends the <a href='http://www.w3.org/TR/hr-time/#idl-def-Performance'>Performance</a> interface [[!HR-TIME-2]] and hosts performance related attributes and methods used to retrieve the performance metric data from the <a>Performance Timeline</a>.</p>
```
Comment (igrigorik) [inline index.html]: aside: we need to updated the published version of HR-TIME.

**Comment by igrigorik** [comment]
lgtm, modulo above knits.

**Comment by toddreifsteck** [comment]
Ilya caught everything. LGTM

File: index.html (hunk 1 above)
Comment (plehegar) [inline index.html]: Argh. My fault. I modify the link while editing and then forget to remove it. Sorry.

File: index.html (hunk 2 above)
Comment (plehegar) [inline index.html]: done for hr-time-2 but further are blocked due to a bug in respec. I need to set some time aside to track it :(

**Comment by igrigorik** [comment]
@plehegar what's the respec bug blocking this? Can we still land this in the meantime, perhaps with a note to address the fix later? The inconsistency with defining the interface in two places is causing lots of confusion.

**Comment by plehegar** [comment]
Finally caught up on this one.

**Pull request merged**